Felienne/hedy
# [TUTORIAL] Adds speak aloud step

**Description**
In this PR we add an additional step to the user tutorial explaining the speak aloud functionality. We add this step between the run explanation and the next level explanation, translations are added for both English and Dutch.

**Fixes**
This PR fixes #2765.

**How to test**
Make sure you are logged in and navigate to `/tutorial`. Verify that there is a new step after the "run step", explaining the speak aloud functionality.
